### DISCUSSION TITLE
Coffee brewing control chart

### DISCUSSION THREAD

**Discussion by EugenePavlikov** [graphefruit/Beanconqueror#527] [Ideas]
Hi,

Do you plan to add a coffee brewing chart, considering that it's possible to enter TDS and extraction? 

![brew-chart-color-sm1 (1).jpg](https://user-images.githubusercontent.com/11786389/236047274-53a0bc58-3368-4744-991a-f99c04929bb6.jpg)

**Comment by graphefruit** [comment]
Hello @EugenePavlikov actually there are no plans to support this kind of graphic.
Adding TDS and get the EY is already possible (even without having a chart).
Maybe with integrating the R2 from DIFluid there will be also a bump to this but actually the benefit of integrating this is to small for the other tickets that are requested

**Reply by graphefruit** [reply]
This has been implemented in one of the latest updates, so this topic can be closed now
